backend/plugin/db/driver.go

```diff
@@ -103,6 +103,8 @@ type ConnectionContext struct {
 	ReadOnly bool
 	// MessageBuffer is used for logging messages from the database server.
 	MessageBuffer []*v1pb.QueryResult_Message
+	// TaskRunUID is set when executing a task run, used to set application_name for connection identification.
+	TaskRunUID *int
```

Comment:
how about passing application_name string directly

Reply:
There is also default application name. We can look into it later once it becomes a problem.